uart_task.c

```diff
@@ -203,7 +204,7 @@ void uart_task(void) {
         uart_write_blocking(UART_ESP32, buffer, length);
     }
 
-    if (tud_cdc_n_available(USB_CDC_FPGA)) {
+    if (tud_cdc_n_available(USB_CDC_FPGA) && !fpga_loopback_active) {
```

Comment:
Now the buffer of the cdc fills up and gets emptied after loopback is disabled. Maybe move the loopback check to around the uart write instead?

Reply:
Yeah, that's on purpose actually. The buffer will have backpressure to USB so there is no overflow and will just get sent to the UART normally once the loopback test is over.